packages/stack/storage/package.json

```
@@ -40,7 +40,8 @@
     "lint": "eslint src/",
     "qa": "npm-run-all lint _typecheck _build",
     "reset": "npx rimraf dist embark-*.tgz package",
-    "solo": "embark-solo"
+    "solo": "embark-solo",
+    "test": "jest"
   },
   "eslintConfig": {
     "extends": "../../../.eslintrc.json"
```

```suggestion
    "extends": [
      "../../../.eslintrc.json",
      "plugin:jest/recommended",
      "plugin:jest/style"
    ]
```

`"eslint-plugin-jest": "22.5.1"` will need to be added to devDeps.  With linting in place for `test/`, there will be a linter error that `sinon` is missing from the dependencies, so `"sinon": "7.4.2"` should be added to devDeps.